tests/module-setup.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
```

Comment:
Thank you SO much for getting rid of this nasty "testing for certain directories to determine where I am" logic!